LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoRaPayloadJoinAccept.cs

```diff
@@ -50,6 +50,10 @@ public class LoRaPayloadJoinAccept : LoRaPayload
         /// </summary>
         public Memory<byte> Fcnt { get; set; }
 
+        public int Rx1DrOffset => (this.DlSettings.Span[0] >> 4) & 0b00000111;

```

Comment:
this comment didn't make it for some reason: what I don't understand is, you right shift by 4 and then mask the first 4 bytes with 01111 which will result to 7 always, unless DlSettings.Span is larger than 8 bytes? Not sure what it is really. Did you mean to left shift?